static/js/reducers/posts.js

```diff
@@ -0,0 +1,15 @@
+// @flow=
```

Comment:
If they're just to provide dummy data I think that's fine for now, as the APIs get into place we can quickly switch over to using `hammock`.

also if you want to read about it there's [documentation](https://github.com/mitodl/redux-hammock) now!